syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.hazelcast.platform.demos.banking.cva";

package com_hazelcast_platform_demos_banking_cva;

service JetToCpp {
    rpc streamingCall (stream InputMessage) returns (stream OutputMessage) {}
}
message InputMessage {
    repeated string inputValue = 1;
}
message OutputMessage {
    repeated string outputValue = 1;
}